runelite-api/src/main/java/net/runelite/api/VarClientInt.java

```diff
@@ -52,6 +52,9 @@
 	 */
 	INPUT_TYPE(5),
 
+	CAMERA_ZOOM_1(73),
+	CAMERA_ZOOM_2(74),
```

Comment:
What are these? Why is only one of them used? What are the expected values?

Reply:
They seem to be the internal camera zoom level. I'm unsure what the difference between both vars are, though - both always seem to be equal, even with the vanilla behaviour.

ScriptID.java:163 also just refers to both of these vars as being the zoom level.
CameraPlugin.keyReleased() doesn't give much idea either, as it simply calls the script with the same value for its two arguments (which presumably map to vars 73 & 74).